src/vuln_analysis/functions/cve_agent.py

```diff
@@ -169,7 +169,7 @@ async def _process_step(step):
         try:
             return await agent.ainvoke({"input": step}, config={"callbacks": callbacks})
         except Exception as e:
-            logger.error(" Error in agent execution : %s", e)
+            logger.error("Error in agent execution: %s", type(e).__name__)
```

Comment:
I think these are missing the error message now?
```suggestion
            logger.error("Error in agent execution: %s: %s", type(e).__name__, e)
```